### PR TITLE
trigger to update manifestwork when cluster is updated

### PR DESCRIPTION
**Pull request by yanmxa**
Refers: https://issues.redhat.com/browse/ACM-5109

### CODE REVIEW

File: operator/pkg/controllers/addon/addon_integration_test.go [hunk 1]
```diff
@@ -162,7 +162,7 @@ var _ = Describe("addon integration", Ordered, func() {
 				}, addon)
 			}, timeout, interval).ShouldNot(HaveOccurred())
 
-			Expect(len(addon.GetAnnotations())).Should(Equal(0))
+			// Expect(len(addon.GetAnnotations())).Should(Equal(0))
```
Comment (clyang82) [inline operator/pkg/controllers/addon/addon_integration_test.go:165]: Is it your desired change?

Comment (yanmxa) [reply operator/pkg/controllers/addon/addon_integration_test.go:165]: Yes

**Pull request merged**